parm/config/config.fcst

```diff
@@ -205,6 +205,7 @@ export deflate_level=1
 
 export OUTPUT_FILETYPES="$OUTPUT_FILE"
 if [[ "$OUTPUT_FILE" == "netcdf" ]]; then
+    export OUTPUT_FILETYPES=" 'netcdf_parallel' 'netcdf' "
```

Comment:
Is there a problem with having `netcdf_parallel` for the surface files?
Unless there is, I suggest the defaults to be `netcdf_parallel`.

Reply:
I'm not sure. That's why I called you and Kate in to help assess these changes. They are almost functionally equivalent to before, but that doesn't mean we actually need these resolution distinctions.

Reply:
So I think I was the one who originally put in the ush/parsing_model_configure_FV3.sh statement that's being taken out here.  I did this to get around issues that I was having when switching to use parallel netcdf in feature/coupled-crow.  Perhaps we wont have those issues in the develop branch now.  (And sorry I can't remember the exact issue).

Reply:
It looked like it was duplicating what was being done in config.fcst (probably that update came after coupled-crow forked from develop, then you added it in a different location since config files work differently in CROW).